### PR TITLE
Specify host before prerender via config file

### LINKED ISSUE #8: OG image broken with switch to default prerendering mode
SvelteKit has a prerender mode, which allows to save on client rendering.
However, this seemed to break the URLs to the OG images: https://www.opengraph.xyz/url/https%3A%2F%2Fwww.hololivefanbooth.com

The solution is somewhere here: https://github.com/watsonindustries/hololivefanbooth/blob/main/src/routes/%2Blayout.svelte#L25

Don't forget about the secure image url too.

### PR DESCRIPTION
**Pull request by CenTdemeern1**
Fixes #8

### CODE REVIEW

**Comment by CenTdemeern1** [comment]
I'll also quickly add that note to the readme

File: src/routes/+layout.ts [hunk 1]
```diff
@@ -1,11 +1,12 @@
 import type { LayoutLoad } from './$types';
+import { host } from '../config.json';
 
 import boothLogoLarge from '$lib/assets/fanbooth-logo-large.png';
 
 export const prerender = true;
 
 export const load = (async ({ url }) => {
```
Comment (danirukun) [inline src/routes/+layout.ts]: Can probably drop 'url' here if it is unused.

**Comment by CenTdemeern1** [comment]
> 😎 Deploy Preview	https://deploy-preview-9--jovial-pastelito-09975c.netlify.app
---
Here is a screenshot of the embed image working on the netlify preview

Screenshot 2023-06-13 at 18 35 15

File: src/routes/+layout.ts (hunk 1 above)
Comment (CenTdemeern1) [reply src/routes/+layout.ts]: Oh yeah true, didn't think about that

Comment (CenTdemeern1) [reply src/routes/+layout.ts]: @DaniruKun does this look alright?

**Review by danirukun — APPROVED** [review]
Danke m!

**Pull request merged**